src/components/input/input-base.ts

```diff
@@ -41,8 +40,7 @@ export abstract class IgcInputBaseComponent
   protected inputId = `input-${nextId++}`;
 
   /** The value attribute of the control. */
-  @blazorSuppress()
-  public abstract value: string;
+  //public abstract value: string;
```

Comment:
I guess you dropped this due to the type changing in the date input, though that can be resolved as well - you just need the base to be a superset of all types and the extending classes to narrow it down.

Also, even like this the even mapping is still coming from the base and the `igcChange` interface is still typed as a string and we need to override that. If we make the base generic on the value type it could likely solve that but still need to check.

Also not sure how that'd affect Blazor analysis